### DISCUSSION TITLE
Can't upgrade\downgrade Python packages (torch / torchvision)  need help

### DISCUSSION THREAD

**Discussion by narzoo** [LykosAI/StabilityMatrix#910] [Q&A] [answered]
Hello everyone!

I have a problem. At some point, in Stability Matrix, I lost the ability to update the torch and torchvision packages. Previously, I could choose the desired version and install it, but now this option has disappeared. They simply do not appear in the list, while this issue did not affect other packages, only torch and torchvision.

Has anyone encountered this, and can you help me? I would be very grateful.

Just so you know, I have the latest version of Stability Matrix as of today, but the problem appeared even before the update. Maybe I need to clear some cache or something... I’m not sure :(

![image](https://github.com/user-attachments/assets/fdc8fa4b-bf47-4bed-a797-3f0fab68b3e7)
![image](https://github.com/user-attachments/assets/84c3cd28-ec5c-42ca-b1b2-ca5af1280fed)

**Comment by Nanamura** [comment]
same here, can't choose version for torch/torchaudion/torchvision , after the last update (also search block crash tha app if u type too fast)

**Comment by Nanamura** [comment]
Found a solution , click "+" button and enter (torch/torchaudio/torchvision) whatever u need to fix. its downloads the latest version of selected object and enable(fix) an option to change its version

**Reply by narzoo** [reply]
Thank you very much for your comment, I tried it, but unfortunately it didn't help. The problem persists... But maybe it will help someone else!

**Comment by narzoo** [comment, marked as answer]
Fixed in the las update  2.12.1 
![image](https://github.com/user-attachments/assets/7c8bde84-d7a3-4af3-bda5-172ede0653d3)
Thank you <3
